v7/latex/README.md

```diff
@@ -0,0 +1,42 @@
+This plugin allows to write posts and pages in a LaTeX-like syntax.
```

Comment:
The idea is that simple enough files (which don't use features the page compiler doesn't support) could be compiled both by the page compiler or being included in a proper LaTeX document and result in the "same" output (as much as HTML and PDF can be the same).